code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm

```diff
@@ -0,0 +1,328 @@
+The king goat pelt: Hope you brought a knife cause your gonna need to butcher the king goats corpse to get this prize. Once you butcher the king goat you can grab his pelt and wear it on your head as armor, boasting complete bomb immunity and slightly better gun and laser immunity then the drake helm at the cost of slightly reduced melee protection this is THE prize to show who the king of lavaland really is around here! Also makes goats friendly towards you as long as you are wearing it for they will see you as their new king.
+Difficulty: Insanely Hard
+*/
+
+//Visager's tracks 'Battle!' and 'Miniboss Fight' from the album 'Songs from an Unmade World 2' are available here
+//http://freemusicarchive.org/music/Visager/Songs_From_An_Unmade_World_2/ and are made available under the CC BY 4.0 Attribution license,
+//which is available for viewing here: https://creativecommons.org/licenses/by/4.0/legalcode
+
+
+//the king and his court
+/mob/living/simple_animal/hostile/retaliate/goat/king
+	name = "king of the goats"
+	desc = "The oldest and wisest of the goats. King of his race, peerless in dignity and power. His golden fleece radiates nobility."
+	icon = 'icons/mob/king_of_goats.dmi'
+	icon_state = "king_goat"
+	icon_living = "king_goat"
+	icon_dead = "king_goat_dead"
+	faction = list("goat_king")
+	attack_same = FALSE
+	speak_emote = list("brays in a booming voice")
+	emote_hear = list("brays in a booming voice")
+	emote_see = list("stamps a mighty foot, shaking the surroundings")
+	response_help_continuous  = "placates"
+	response_harm_continuous   = "assaults"
+	attack_verb_simple = "brutalized"
+	health = 500
+	a_intent = INTENT_HARM
+	sentience_type = SENTIENCE_BOSS
+	stat_attack = DEAD
+	wander = FALSE
+	maxHealth = 500
+	armour_penetration = 35
+	melee_damage_lower = 35
+	melee_damage_upper = 55
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	obj_damage = 400
+	vision_range = 5
+	aggro_vision_range = 18
+	move_force = MOVE_FORCE_OVERPOWERING
+	move_resist = MOVE_FORCE_OVERPOWERING
+	pull_force = MOVE_FORCE_OVERPOWERING
+	mob_size = MOB_SIZE_LARGE
+	robust_searching = TRUE
+	move_to_delay = 3
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+
+
+
+	var/stun_chance = 5 //chance per attack to Weaken target
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/ex_act(severity, target)
+	switch (severity)
+		if (1)
+			adjustBruteLoss(100)
+
+		if (2)
+			adjustBruteLoss(50)
+
+		if(3)
+			adjustBruteLoss(25)
+
+/mob/living/simple_animal/hostile/retaliate/goat/king/phase2
+	name = "emperor of the goats"
+	desc = "The King of Kings, God amongst men, and your superior in every way."
+	icon_state = "king_goat2"
+	icon_living = "king_goat2"
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 36, /obj/item/clothing/head/goatpelt/king = 1)
```

Comment:
Is 36 slabs of meat really needed? That just seems excessive to the point where it appears to be creating item spam for no reason.